src/args/calculate-cvss-3/cvss.py

```diff
@@ -17,7 +21,7 @@ def main(vector: str) -> None:
     scores: Tuple[str, str, str] = _str(result.scores())
     severities: Tuple[str, str, str] = result.severities()
 
-    print(
+    LOGGER.info(
```

Comment:
Does this add any type of prefix to the string like `[INFO]`?

Also, when doing this in an interactive shell, it looks like logger does not return anything:
![image](https://github.com/fluidattacks/makes/assets/11205987/a3214e99-b1b7-4de2-8e38-63f799553b51)

This script must print the output to stdout without any prefix or suffix.

Reply:
Yes, I've been using it sometimes to debug on `streams` at `universe` and it shows like that. I don't know why it isn't showing the string test.